Vizzuality/half-earth-v3
# Fix/nrc-onboarding

## NRC onboarding tweaks and Trend chart only max value
### Description
Remove some part of the onboarding and change order to make sense to the new NRC page
Add a disclaimer on the home
Use only max value on the Trend Chart (Lines no areas)
### Designs
Disclaimer: https://www.figma.com/file/2YAp3AS5lYCj2IWEAN8AJw/HE-Map---Work-in-Progress?node-id=0%3A1&t=muXNb8RQJf4bVy4q-1
### Testing instructions
Go to the NRC onboarding. Test it. 
Go to the NRC page. Check the Chart to see the lines instead of Areas
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-670?atlOrigin=eyJpIjoiMzg2ZmMxOTZlOTJjNGM3ZWJlNjkxOTA1ODlhNDk2NTAiLCJwIjoiaiJ9

https://vizzuality.atlassian.net/browse/HE-671?atlOrigin=eyJpIjoiNjhmNzBhYjIxMThmNDc1YWFiZDU3ODJjZTUzY2Q0ZDUiLCJwIjoiaiJ9
